inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/dialect/PostgresDialect.java

```diff
@@ -169,7 +169,7 @@ public List<LogicalTypeRoot> unsupportedTypes() {
     public PreparedStatement setQuerySql(Connection conn,
             String tableIdentifier) throws SQLException {
```

Comment:
The method name is too generic. It can't express specific meaning.

Reply:
done